### ISSUE TITLE
How to obtain validation_raw subfolders without complete all epochs training?

### ISSUE THREAD

**Issue by DomenicoMessina** [MIC-DKFZ/nnUNet#576]
Hello, 

I congratulate you on the excellent work done with nnUNet. It is a fantastic framework.
I'm training a task with nnUNet, but I can't train the net for all the preset epochs (1000 epochs) for time limits. I partially trained all five folds for the task. 
Is there a way to move to the next stage of the pipeline without run all 1000 epochs for all folds?  Could I try to change the parameter on some plans file?

Sorry for such a basic question.
Regards,
Domenico

**Comment by FabianIsensee** [comment]
This is not recommended because the learning rate schedule needs the whole 1000 to be effective. Training times are also not that long (anymore): You can train one nnU-net model in less than 2 days (TitanRTX, V100) or even less than one day (A100).

If you still want to proceed, the way to do this would be to rename the checkpoints (use latest) to `model_final_checkpoint.model(.pkl)`. Then simply use `nnUNet_train [...] -val`

**Comment by DomenicoMessina** [comment]
Thank you, Fabian, for the fast and exhaustive answer. I'm closing this issue
